cpu/riscv_common/irq_arch.c

```diff
@@ -149,8 +151,7 @@ static void __attribute((used)) handle_trap(uint32_t mcause)
 /* Marking this as interrupt to ensure an mret at the end, provided by the
  * compiler. Aligned to 64-byte boundary as per RISC-V spec and required by some
  * of the supported platforms (gd32)*/
-static void __attribute((aligned(64))) __attribute__((interrupt)) trap_entry(void)
-{
+static void __attribute((aligned(64))) __attribute__((interrupt)) trap_entry(void){
```

Comment:
same here

Reply:
Hmm, like so?

```suggestion
__attribute((aligned(64)))
static void __attribute__((interrupt)) trap_entry(void){
```

(I am not entirely sure about the difference between `__attribute__`s and `__attribute__`s yet.)